test/quicer_connection_SUITE.erl

```diff
@@ -462,6 +462,86 @@ tc_conn_client_bad_cert(Config) ->
       ct:fail({run_error, Error})
   end.
 
+tc_datagram_disallowed(Config) ->
+  Port = select_port(),
+  ServerConnCallback = example_server_connection,
+  ServerStreamCallback = example_server_stream,
+  ListenerOpts = [{conn_acceptors, 4} | default_listen_opts(Config)],
+  ConnectionOpts = [ {conn_callback, ServerConnCallback}
+                   , {stream_acceptors, 2}
+                     | default_conn_opts()],
+  StreamOpts = [ {stream_callback, ServerStreamCallback}
+               , {disable_fpbuffer, true}
+               | default_stream_opts() ],
+  %% GIVEN: A listener with datagram_receive_enabled = false
```

Comment:
Q: should `disable_fpbuffer = true` above be `datagram_receive_enabled = false`?
Or maybe explicitly set it in `ListenerOpts`?